pyrokinetics/kinetics/jetto.py

```diff
@@ -8,67 +8,59 @@
 # Can't use xarray, as JETTO has a variable called X which itself has a dimension called X
 import netCDF4 as nc
 import numpy as np
+from jetto_tools.binary import read_binary_file
 
 
 class KineticsReaderJETTO(KineticsReader):
-    impurity_charge_to_mass = dict(
-        zip(
-            [2, 6, 8, 10, 18, 54, 74],
-            [4, 12, 16, 20, 40, 132, 184],
-        )
-    )
-
-    impurity_mass_to_charge = {
-        value: key for key, value in impurity_charge_to_mass.items()
-    }
-
     def read(
         self, filename: PathLike, time_index: int = -1, time: float = None
     ) -> Dict[str, Species]:
         """
         Reads in JETTO profiles NetCDF file
         """
         # Open data file, get generic data
-        with nc.Dataset(filename) as kinetics_data:
-            time_cdf = kinetics_data["TIME3"][:]
+        try:
+            kinetics_data = read_binary_file(filename)
+            jetto_jss = read_binary_file(str(filename).replace("jsp", "jss"))
+            time_cdf = kinetics_data["TIME"].T[:]
```

Comment:
I don't think want to remove the ability to read the `netCDF` file, just add in the ability to read `.jsp` and `.jss`. Ideally we would be able to read both `jsp` and `netCDF` files and then also require there to be the `.jss` file to get the charge information

Reply:
@fcasson can probably comment on this best

Reply:
Personally I would prefer it if you just read the jsp files.  It is is closer to the single source of truth.  The NETCDF is not alays present and represents an extra post-processing step.

Reply:
Okay happy to make the switch, we'll need to switch out the template files too and find the corresponding `.jsp` and `.jss` files

Reply:
@bpatel2107 Could you switch the template file?